dockerfile/ci-tasks/overlays.repo

```diff
@@ -0,0 +1,11 @@
+[rhinstaller]
+name=Copr repo for rhinstaller daily builds for RHEL 8
+baseurl=http://coprbe.devel.redhat.com/results/rhinstaller-group/Anaconda/rhel-8.dev-$basearch/
+enabled=1
+
+[cobra2-rhel-fallback]
+name=Cobra02 custom repository used to solve temporary missing new versions
+baseurl=http://cobra02.anaconda.englab.brq.redhat.com/trees/tests_fallback_repos/rhel8/
```

Comment:
Is it fine to have these links here?

Reply:
If you google for them, there are several hits in bugzilla and fedorapeople that already expose them. There's a lot more hits for the download.devel.redhat.com repos. We also expose them in Cockpit bots, etc. Their names are not secret really, they don't expose anything about customers, projects, etc. That's why I wouldn't worry too much about that -- the benefits for making it really easy to build/use these containers outweigh that IMHO.